core/reflection.jvm/src/kotlin/reflect/jvm/internal/calls/InlineClassAwareCaller.kt

```diff
@@ -96,6 +93,11 @@ internal class InlineClassAwareCaller<out M : Member?>(
 
             descriptor.valueParameters.mapTo(kotlinParameterTypes, ValueParameterDescriptor::getType)
         }
+
+        // If the default argument is set,
+        // (kotlinParameterTypes.size + Integer.SIZE - 1) / Integer.SIZE masks and one marker are added to the end of the argument.
+        val extraArgumentsTail = (if (isDefault) ((kotlinParameterTypes.size + 31) / Integer.SIZE) + 1 else 0) +
```

Comment:
```suggestion
        val extraArgumentsTail = (if (isDefault) ((kotlinParameterTypes.size + Integer.SIZE) / Integer.SIZE) + 1 else 0) +
```
Minor, but let's either use 31/32 in both places, or `Integer.SIZE` in both places. :)